README.md

```diff
@@ -1 +1,44 @@
-Welcome to the port of Jabbr to ASP.NET Core
+## Welcome to the port of JabbR to ASP.NET Core
+---
+This is an open source, cross platform version of David Fowler's [JabbR](https://github.com/JabbR/JabbR) running on .NET Core.
+
+## Setting up for local development
+ For security, connection strings and other sensitive information must be stored in environment variables.
+ - You can easily read and write these key-value pairs using the `dotnet user-secrets` command
+   - You must first add a reference to `Microsoft.Extensions.Configuration.UserSecrets` in your `project.json`
+ - This command must be executed from the top level of your ***project*** (read: not solution). This is
+   because it will try to find a `project.json` file referencing the executable.
+   - You can easily test if the executable is found by typing 
+   ```
+   $ dotnet user-secrets -h
+   ```
+
+ If you correctly followed the above steps, you'll be met with a help dialog when you execute
+ `dotnet user-secrets -h` that looks like this. ![dotnet user-secrets help](https://i.gyazo.com/9220f055788ff9bb6cd24da9a14e6076.png)
+
+ If you receive an error like the following
+ ```bash
+$ dotnet user-secrets -h
+No executable found matching command "dotnet-user-secrets"
+ ```
+
+ Then you likely need to rebuild your solution, or move into the correct project directory where the `project.json` file can be found.
+
+To set a user secret, you need to use `dotnet user-secrets set <key> <value>`, like so
+
+```bash
+$ dotnet user-secrets set "connectionString" "Server=MYAPPNAME.database.windows.net,1433;Initial Catalog=MYCATALOG;Persist Security Info=False;User ID={plaintext user};Password={plaintext password};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
```

Comment:
lets add in some directions to specifically give them the exact commands needed to run locally, ie localdb.